controller/backing_image_controller.go

```
@@ -310,22 +345,604 @@
+
+		delete(bi.Status.DiskFileStatusMap, v2DiskUUID)
+		bic.v2CopyBackoff.DeleteEntry(v2DiskUUID)
+		bic.eventRecorder.Eventf(bi, corev1.EventTypeNormal, constant.EventReasonDelete, "Deleted failed v2 backing image %v on disk %v on node %v", bi.Name, v2DiskUUID, node.Name)
+	}
+	return nil
+}
+
+func (bic *BackingImageController) cleanupAllV2BackingImageCopies(bi *longhorn.BackingImage) (cleaned bool, err error) {
+	log := getLoggerForBackingImage(bic.logger, bi)
+
+	if bic.syncV2StatusWithInstanceManager(bi); err != nil {
```

golangci-lint (1.62.2)

Error return value of `bic.syncV2StatusWithInstanceManager` is not checked

(errcheck)